spec.md

```diff
@@ -556,9 +564,9 @@ The `code` field MUST be one of the following:
 | code-2  | `BLOB_UPLOAD_INVALID`   | blob upload invalid                            |
 | code-3  | `BLOB_UPLOAD_UNKNOWN`   | blob upload unknown to registry                |
 | code-4  | `DIGEST_INVALID`        | provided digest did not match uploaded content |
-| code-5  | `MANIFEST_BLOB_UNKNOWN` | blob unknown to registry                       |
+| code-5  | `MANIFEST_BLOB_UNKNOWN` | manifest references a blob unknown to registry |
```

Comment:
Clarifying that this error code applies to unknown manifest digests in addition to blob digests.

```suggestion
| code-5  | `MANIFEST_BLOB_UNKNOWN` | manifest references a manifest or blob unknown to registry |
```

Reply:
Thanks, change added.